simulariumio/data_objects/camera_data.py

```diff
@@ -57,38 +58,35 @@ def __init__(
         self.fov_degrees = fov_degrees
 
     @classmethod
-    def from_buffer_data(cls, buffer_data: Dict[str, Any]):
-        """ """
-        camera_default = (
-            buffer_data["trajectoryInfo"]["cameraDefault"]
-            if "cameraDefault" in buffer_data["trajectoryInfo"]
-            else None
+    def from_dict(cls, buffer_data: Dict[str, Any]):
+        """
+        Create CameraData object from a simularium JSON dict
+        """
+        # dictionary keys may be in camel case or snake case
+        camera_default = buffer_data.get(
+            "cameraDefault", buffer_data.get("camera_defaults")
```

Comment:
probably don't need the `camera_defaults`?

Reply:
oops yep missed that one

Reply:
or actually, shouldn't the buffer_data argument be the `cameraDefault` module already?